### PR TITLE
Credstash as secret management solution

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
WIP example of how credstash (https://github.com/fugue/credstash) could be used as a secret management solution

### How to review 
With EQ_DEV_KEYS set to False the application should reach out to DynamoDB and AWS KMS for its secrets

### CODE REVIEW

**Comment by collisdigital** [comment]
Nice, what needs to be setup in AWS to enable this to work?

**Comment by jonnyshaw89** [comment]
@collisdigital a KMS alias and a DynamoDB table
And obviously the keys loaded in.

**Comment by jonnyshaw89** [comment]
@collisdigital was also talking to @ajmaddaford (MadDog) and this could also be used for rolling our encryption keys as they secrets can have versions and are immutable

**Comment by collisdigital** [comment]
👌 ; would be good to compare with Vault; but looks promising.

**Comment by jonnyshaw89** [comment]
@collisdigital the issue i can see with Vault is that we would need to deploy our own instances with cluster and autoscaling.
Which is an option, but also its a single point of failure.

File: config.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ class Config(object):
 
 class Test(Config):
     DEBUG = True
-    SECRET_KEY = settings.EQ_SECRET_KEY
```
Comment (collisdigital) [inline config.py]: Not sure if this whole file is even used/needed?

Comment (jonnyshaw89) [reply config.py]: Apparently not. Ill delete it.

**Pull request closed without merging**